packages/esm-billing-app/src/helpers/functions.ts

```diff
@@ -81,3 +81,15 @@ function removeUUID(str) {
   // Replace the UUID with an empty string
   return str.replace(uuidPattern, '');
 }
+
+export const getErrorMessage = (err: any) => {
+  if (err.response?.data.message) {
+    return err.response.data.message;
+  }
+
+  if (err.message) {
+    return err.message;
+  }
+
+  return 'an unknown error occurred';
```

Comment:
Is it possible to have translation for this

```suggestion
  return t('unKnownErrorMsg','An unknown error occurred')
```
```suggestion
  return 'an unknown error occurred';
```

Reply:
I am still familiarizing yourself with how you guys are doing translations but I assume you guys no better so I`m going to committ this change